gemini/getting-started/intro_gemini_pro_python.ipynb

```diff
@@ -0,0 +1,650 @@
+{
```

Comment:
<p>Update the licence year.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>